### DISCUSSION TITLE
Notification about new version of node-red-contrib-home-assistant-websocket

### DISCUSSION THREAD

**Discussion by paulius2k** [zachowj/node-red-contrib-home-assistant-websocket#1047] [Q&A]
What's the best way to get notified about the new version of `node-red-contrib-home-assistant-websocket` released?

**Comment by zachowj** [comment]
Watch this repo with releases selected.

**Reply by zachowj** [reply]
![image](https://github.com/zachowj/node-red-contrib-home-assistant-websocket/assets/37859597/4a6b9a1a-42b5-41bc-a95a-ace59223a044)
